docs/pages/installation.rst

```diff
@@ -23,6 +23,33 @@ Install the package from the ``conda-forge`` channel:
 
     conda install dask-sql -c conda-forge
 
+Experimental GPU support
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+- GPU support is currently  tied to the `RAPIDS <https://rapids.ai/>`_  libraries.
+- It is in development and generally requires the latest cuDF nightlies.
+- It is experimental, so users should expect some bugs or undefined behavior.
+
+Create a new conda environment or use an existing one to install RAPIDS with the chosen methods and packages:
+
+.. code-block:: bash
+
+    conda create --name rapids-env -c rapidsai-nightly -c nvidia -c conda-forge -y cudatoolkit=11.2 cudf=21.12 ucx-py ucx-proc=*=gpu
```

Comment:
Adding the dask-cudf requirement I mentioned above; also fairly sure that we now require CUDA 11.5 for cuDF:

```suggestion
    conda create --name rapids-env -c rapidsai-nightly -c nvidia -c conda-forge \
        cudf=22.02 dask-cudf=22.02 ucx-py ucx-proc=*=gpu python=3.8 cudatoolkit=11.5
```